nflgame/game.py

```diff
@@ -336,11 +365,11 @@ def save(self, fpath=None):
         if fpath is None:
             fpath = _jsonf % self.eid
         try:
-            print >> gzip.open(fpath, 'w+'), self.rawData,
+            print(self.rawData, file=gzip.open(fpath, 'w+'))
```

Comment:
```suggestion
            with gzip.open(fpath, 'w+') as outfile:
                outfile.write(self.rawData)
```
gzip complains about expecting bytes and receiving str without this fix.

Reply:
@lynshi your suggestion works for me on Python 3.7.2